### DISCUSSION TITLE
command not found: deta

### DISCUSSION THREAD

**Discussion by mahmudbabakusa** [deta/cloud-docs#643] [Help]
After successfull installation of deta unsing the terminal on mac but still gives me command not found: deta. Even as I activate it. Please how can I set it up on my mac so that the command becomes globally. A special thanks to the deta team and community.

**Comment by jnsougata** [comment]
Can you please specify your OS?

**Reply by mahmudbabakusa** [reply]
Mac Os

**Reply by mahmudbabakusa** [reply]
macOS Ventura 13.0.1

**Comment by joakm-bang** [comment]
I have the same issue. I'm running MacOS 12.6.1

![deta](https://user-images.githubusercontent.com/18540391/206923508-8931588b-b37f-484d-9071-04f93cdb90ec.png)

**Comment by lemonyte** [comment]
Check if you have the line `export PATH="/Users/<username>/.deta/bin:$PATH"` (or similar) in your `.zshrc` or `.bashrc` file.

**Reply by iamumairayub** [reply]
I have `export PATH="/root/.deta/bin:$PATH"` in my `.bashrc` but still I am getting `deta: command not found` error in my Ubuntu Linux.
